CryptoLightNews/Model/NewsArticles.swift

```diff
@@ -7,7 +7,7 @@
 
 import Foundation
 
-struct NewsArticles: Codable, Equatable {
+struct NewsArticles: Codable {
```

Comment:
Here is more complicated to decide: you can have a more generic `Article` or be more precise and name it `NewsArticle`; but always in singular.
```suggestion
struct Article: Codable {
```